AbilityDeck.jsx

```diff
@@ -86,22 +103,105 @@ export default function AbilityDeck(props) {
     );
   };
 
+  const renderBaseStats = () => {
+    if (!props.showBaseStats) {
+      return null;
+    }
+
+    const statLine = (iconName, data) => {
+      const bossData = (data.length === 1);
+      const empty = (data[0] === 0) && (bossData || (data[1] === 0));
+      if (empty) {
+        return null;
+      }
+
+      const value = bossData ? String(data[0]) : `${data[0]}/${data[1]}`;
```

Comment:
For 0 values, it might look nicer to render them as "-" instead? Also, was chatting with adrake and he suggested maybe handling the non-stat attributes as lists of the form `N: <img .../>,<img .../>, <img src=shield.svg/> 2` and `E: <img .../>,<img .../>, <img src=shield.svg/> 2`.  What do you think?

Reply:
re: - instead of 0, sure, np.
re: boolean stats; yes I had considered doing this originally.  What I really wanted to do was to show the icons in either black for normals, and gold for elites.  This sent me down a long path trying to figure out why it was so difficult to do something like a chromakey filter on an SVG (to change the white in the icon to gold).  I eventually decided that "it's complicated" and that I should put something like that off to a future CL.

I also considered splitting the elite and the normal stats, putting one on the left side of the deck and the other on the right.  This way, I could just list all of the numeric stats and then have one line at the bottom for the bool stats.  In the end I though that this would just take too much room, and I would still want to have solved the color coding problem before trying it.

Another option I toyed with was rendering bool stats as "no/yes" instead of "0/1".  In the end, I decided to try to avoid feature creep and just start with this, assuming that it could be changed moving forward.

I'll rework this for now so that we can show them as two sets of bool stats; one for normals and one for elites.  I think it would probably look better with color coding, but I'd like to leave that as a future project for now.